MAUI/ListView/viewappearance.md

```diff
@@ -0,0 +1,1208 @@
+        }
+    }
+
+    private async void GenerateItems()
+    {
+        IsLoading = true;
+        await Task.Delay(5000);
+        for (int i = 0; i < 30; i++)
+        {
+            var contact = new Model(CustomerNames[i], ContactNumber[i]);
+            ContactInfo.Add(contact);
+        }
+        IsLoading = false;
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> When both `SfBusyIndicator` and `SfListView` loaded with same row and column, you need to set `InputTransparent` as `True` to `SfBusyIndicator` in order to pass touch interaction to `SfListView` in iOS platform.
+
+## show busy indicator on list view items
+
+The `SfListView` allows displaying an activity indicator for an item when its data is being loaded in the background. To perform this, load both `ActivityIndicator` and a `Button` in the same row of a `Grid` element inside the [ItemTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_ItemTemplate) of the `SfListView`. The busy indicator and button can be enabled and disabled by using properties `IsButtonVisible` and `IsIndicatorVisible` respectively in the model class.
+
+{% tabs %}
+{% highlight c# %}
+public class BookInfo : INotifyPropertyChanged
+{
+    private string bookName;
+    private string bookDescription;
+    public bool isDescriptionVisible;
+    public bool isButtonVisible;
+    public bool isIndicatorVisible;
+
+    public BookInfo()
+    {
+    }
+
+    public string BookName
+    {
+        get { return bookName; }
+        set
+        {
+            bookName = value;
+            OnPropertyChanged("BookName");
+        }
+    }
+
+    public bool IsDescriptionVisible
+    {
+        get { return isDescriptionVisible; }
+        set
+        {
+            isDescriptionVisible = value;
+            OnPropertyChanged("IsDescriptionVisible");
+        }
+    }
+
+    public string BookDescription
+    {
+        get { return bookDescription; }
+        set
+        {
+            bookDescription = value;
+            OnPropertyChanged("BookDescription");
+        }
+    }
+
+    public bool IsButtonVisible
+    {
+        get { return isButtonVisible; }
+        set
+        {
+            isButtonVisible = value;
+            OnPropertyChanged("IsButtonVisible");
+        }
+    }
+
+    public bool IsIndicatorVisible
+    {
+        get { return isIndicatorVisible; }
+        set
+        {
+            isIndicatorVisible = value;
+            OnPropertyChanged("IsIndicatorVisible");
+        }
+    }
+
+    public event PropertyChangedEventHandler PropertyChanged;
+
+    public void OnPropertyChanged(string name)
+    {
+        if (this.PropertyChanged != null)
+            this.PropertyChanged(this, new PropertyChangedEventArgs(name));
+    }
+}
+{% endhighlight %}
+{% endtabs %}
+
+Disable the visibility of Description and `ActivityIndicator` initially while adding items into collection.
```

Comment:
initially, disable the visibility of description and `ActivityIndicator` when adding items to a collection.

Reply:
I have modified the line as per the concerns provided.